tests/Console/ConsoleEventSchedulerTest.php

```diff
@@ -24,20 +27,22 @@ public function testExecCreatesNewCommand()
         $this->assertEquals('path/to/command', $events[0]->command);
         $this->assertEquals('path/to/command -f --foo="bar"', $events[1]->command);
         $this->assertEquals('path/to/command -f', $events[2]->command);
-        $this->assertEquals('path/to/command --foo=\'bar\'', $events[3]->command);
-        $this->assertEquals('path/to/command -f --foo=\'bar\'', $events[4]->command);
-        $this->assertEquals('path/to/command --title=\'A "real" test\'', $events[5]->command);
+        $this->assertEquals('path/to/command --foo='.$escape.'bar'.$escape, $events[3]->command);
+        $this->assertEquals('path/to/command -f --foo='.$escape.'bar'.$escape, $events[4]->command);
```

Comment:
Maybe you mean replaces `'.$escape.'` to `{$escape}`?

I did concat because it's the original style, for instance:
`$this->assertEquals($binary.' artisan queue:listen --tries=3', $events[2]->command);`

Here is used concat instead of interpolation.

But, i'll do that.